_docs_integrate/version-5.md

```diff
@@ -0,0 +1,130 @@
+---
+# Start automatic generation
+permalink: integrate/version-5
+published: false
+title: "Version 5"
+type: scenario
+toc: true
+properties:
+  - id: SC115
+  - category: Migration Guides
+  - description: Änderungen bzgl. Version 5
```

Comment:
german description?

Reply:
I change this, however in the UseCaseV5 there was something in german, when I filtered so I was confused.

Reply:
https://jssoftgmbh.sharepoint.com/:x:/r/sites/Enmeshed/_layouts/15/Doc.aspx?sourcedoc=%7B485D26E0-DA69-45D3-A25E-922CD9CA8290%7D&file=UseCasesV5.xlsx&wdOrigin=TEAMS-MAGLEV.p2p_ns.rwc&action=default&mobileredirect=true

Reply:
I see.